repos/rust-lang/miri.toml

```diff
@@ -0,0 +1,11 @@
+org = "rust-lang"
+name = "miri"
+description = "An interpreter for Rust's mid-level intermediate representation"
+bots = ["rustbot", "bors"]
+
+[access.teams]
+compiler = "write"
+miri = "write"
+
+[[branch-protections]]
+pattern = "master"
```

Comment:
So which CI job does this check for then?

Reply:
It doesn't do any checks, it just disallows direct pushes to `master`. And yeah, with bors the branch/CI job configuration is managed by bors instead.